### DISCUSSION TITLE
Suspension of My GitHub Account

### DISCUSSION THREAD

**Discussion by ghost** [community/community#135410] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi everyone,

I hope this message finds you well. I'm reaching out because my primary GitHub account, which I rely on heavily for all my professional work, was unexpectedly suspended. This account is critical to my job, and the suspension is causing significant disruption to my ongoing projects.

Account Name: @MoSalem149
Ticket ID: 2924055

I have already submitted a support ticket to GitHub but have not received a response yet. I would greatly appreciate any advice, assistance, or insights from the community that might help expedite the resolution of this issue.

Thank you for your understanding and support during this challenging time.

Best regards,
Mohamed Wael Salem

**Comment by everforyou** [comment]
Hello @MoSalem149v2 
First, Look for any emails from GitHub regarding the suspension. Sometimes, they provide details or reasons for the action.
One more thing, Sometimes reaching out via GitHub's official Twitter account can expedite your request.

**Reply by ghost** [reply]
Thank you for the advice. Unfortunately, I haven’t received any emails from GitHub regarding the suspension, so I’m not sure why it happened. I’ve also tried reaching out via Twitter but haven’t gotten a response yet. Any further suggestions would be greatly appreciated.

**Reply by everforyou** [reply]
Sorry, I can't help. please try to contact with github support team. And 2nd method is fine too.

**Reply by ghost** [reply]
No worries, thank you for your suggestion. I’ve already tried contacting GitHub Support and reached out on Twitter as well. I’m still waiting for a response, but I’ll keep trying. Thanks again for your time.

**Reply by still-hash-ir** [reply]
I am facing the same issue. I started a discussion here (https://github.com/orgs/community/discussions/134510) and also tweeted about it (https://x.com/_hash_ir_/status/1821209794960564227) but no response whatsoever.

**Reply by ghost** [reply]
What to do ? if the account is lost forever or what

**Reply by dmxz-cripto** [reply]
hello

**Comment by deviljerry** [comment]
couple of days ago my other github accont has been suddenly disappeared  i am trying to loggin through that account but its says email and password is wrong can someone also faced that problem and can anyoe tell me the solution for that because my that account contains all my projecs i made in my university life can any other persone faced that problem. Also plss guide me how culd i know that my github account has been suspended i mean if that case happen how could  i know

**Reply by ghost** [reply]
I'm really sorry to hear that your GitHub account disappeared. That’s incredibly stressful, especially with all your university projects on the line.

Here’s what you can try:

Check Your Email: If GitHub suspended your account, they usually send an email. Check your inbox and spam folder for anything from them. If there’s nothing, it might be another issue.

Reset Your Password: If it says your email and password are wrong, try resetting your password. Sometimes that’s all it takes.

Contact Support: If you’re still stuck, reach out to GitHub Support through their Support page. They should be able to help you out.

I hope this helps, and that you get your account back soon. Hang in there!

**Comment by ebndev** [comment, marked as answer]
Hi @MoSalem149v2,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
